src/java.base/share/classes/java/lang/foreign/MemorySegment.java

```diff
@@ -1284,6 +1284,14 @@ MemorySegment reinterpret(long newSize,
      * sequences with this charset's default replacement string. The {@link
      * java.nio.charset.CharsetDecoder} class should be used when more control
      * over the decoding process is required.
+     * <p>
+     * Getting a String from a {@code segment} with a known byte {@code offset} and
+     * known byte {@code length} can be done like so:
+     * {@snippet lang=java
```

Comment:
A semicolon at the end of the line is missing - this is causing build issues in the github action:

```
/home/runner/work/jdk/jdk/src/java.base/share/classes/java/lang/foreign/MemorySegment.java:1291: error: unexpected content
     *     byte[] bytes = new byte[length];
              ^
```